http4s/src/main/scala/com/avast/cloud/metrics/datadog/http4s/DatadogMetricsOps.scala

```diff
@@ -0,0 +1,63 @@
+package com.avast.cloud.metrics.datadog.http4s
+
+import java.time.Duration
+
+import cats.syntax.flatMap._
+import cats.effect.Sync
+import com.avast.cloud.metrics.datadog.api.{ MetricFactory, Tag }
+import org.http4s.{ Method, Status }
+import org.http4s.metrics.{ MetricsOps, TerminationType }
+
+object DatadogMetricsOps {
+
+  type ClassifierTags = String => List[Tag]
```

Comment:
what is a classifier? is that some http4s concept?

Reply:
It's `Option[String]` that is extracted from `Request[F]` by a user provided function.